modules/cnf-programming-the-sriov-operator.adoc

```diff
@@ -0,0 +1,398 @@
+DL-Q'42 |     |  X  |     |     |     |     |     |     |     |
+DL-Q'43 |     |  X  |     |     |     |     |     |     |     |
+DL-Q'44 |     |  X  |     |     |     |     |     |     |     |
+DL-Q'45 |     |  X  |     |     |     |     |     |     |     |
+DL-Q'46 |     |  X  |     |     |     |     |     |     |     |
+DL-Q'47 |     |  X  |     |     |     |     |     |     |     |
+DL-Q'48 |     |     |  X  |     |     |     |     |     |     |
+DL-Q'49 |     |     |  X  |     |     |     |     |     |     |
+DL-Q'50 |     |     |  X  |     |     |     |     |     |     |
+DL-Q'51 |     |     |  X  |     |     |     |     |     |     |
+DL-Q'52 |     |     |  X  |     |     |     |     |     |     |
+DL-Q'53 |     |     |  X  |     |     |     |     |     |     |
+DL-Q'54 |     |     |  X  |     |     |     |     |     |     |
+DL-Q'55 |     |     |  X  |     |     |     |     |     |     |
+DL-Q'56 |     |     |  X  |     |     |     |     |     |     |
+DL-Q'57 |     |     |  X  |     |     |     |     |     |     |
+DL-Q'58 |     |     |  X  |     |     |     |     |     |     |
+DL-Q'59 |     |     |  X  |     |     |     |     |     |     |
+DL-Q'60 |     |     |  X  |     |     |     |     |     |     |
+DL-Q'61 |     |     |  X  |     |     |     |     |     |     |
+DL-Q'62 |     |     |  X  |     |     |     |     |     |     |
+DL-Q'63 |     |     |  X  |     |     |     |     |     |     |
+--------+-----+-----+-----+-----+-----+-----+-----+-----+-----+
+
+Mode of operation = VF-mode
+FPGA_5GNR PF [0000:1d:00.0] configuration complete!"}
+2020-12-16T12:46:48.017Z        INFO    daemon.NodeConfigurator.enableMasterBus executing command       {"cmd": "/usr/sbin/chroot /host/ setpci -v -s 0000:1d:00.0 COMMAND"}
+2020-12-16T12:46:48.037Z        INFO    daemon.NodeConfigurator.enableMasterBus commands output {"output": "0000:1d:00.0 @04 = 0102\n"}
+2020-12-16T12:46:48.037Z        INFO    daemon.NodeConfigurator.enableMasterBus executing command       {"cmd": "/usr/sbin/chroot /host/ setpci -v -s 0000:1d:00.0 COMMAND=0106"}
+2020-12-16T12:46:48.054Z        INFO    daemon.NodeConfigurator.enableMasterBus commands output {"output": "0000:1d:00.0 @04 0106\n"}
+2020-12-16T12:46:48.054Z        INFO    daemon.NodeConfigurator.enableMasterBus MasterBus set   {"pci": "0000:1d:00.0", "output": "0000:1d:00.0 @04 0106\n"}
+2020-12-16T12:46:48.160Z        INFO    daemon.drainhelper.Run()        worker function - end   {"performUncordon": true}
+----
+
+. Check the FEC configuration of the card.
+
++
+[source,terminal]
+----
+$ oc get sriovfecnodeconfig node1 -o yaml
+----
++
+.Example output
+[source,yaml]
+----
+status:
+    conditions:
+    - lastTransitionTime: "2020-12-15T17:19:37Z"
+      message: Configured successfully
+      observedGeneration: 1
+      reason: ConfigurationSucceeded
+      status: "True"
+      type: Configured
+    inventory:
+      sriovAccelerators:
+    - deviceID: 0d8f <1>
```

Comment:
indentation issue the `-` should start as `sriovAccelerators` all the lines below should move accordingly